### PR TITLE
Update Readme-FA

### PR DESCRIPTION
**Pull request by nejads**
Improved language problem in the Persian translation.

### CODE REVIEW

**Comment by raisedadead** [comment]
Hey can you get someone who speak the language to peer review this before we can merge this? Thank you.

**Comment by nejads** [comment]
I will try to do that.

File: README-FA.md [hunk 1]
```diff
@@ -17,27 +17,27 @@
...
 
-# تازه واردین مشارکت کننده در منبع باز خوش آمدید.
+# تازه‌کارهای مشارکت کننده در منبع باز به اینجا خوش آمدید.
 
 [![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](http://makeapullrequest.com)
 [![first-timers-only Friendly](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](http://www.firsttimersonly.com/)
 [![Build Status](https://travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source.svg?branch=master)](https://travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source)
 
-این فهرستی از منابع برای افرادی است که جدیدا تصمیم به کمک منبع باز گرفته اند.
+این فهرستی از منابع و لینک‌هایی‌ست برای افرادی است که جدیدا تصمیم به کمک "منبع باز" گرفته اند.
```
Comment (sam-hosseini) [inline README-FA.md]: I think since the tone of the document is formal, `لینک‌هایی‌ست` would read better with `لینک‌هایی‌ است برای افرادی که`.

**Review by sam-hosseini — COMMENTED** [review]
-

**Comment by erictleung** [comment]
@nejads whenever you get a chance, please make the suggested changes from above, then I can merge them into our project 👍  Let me know if you have any questions.

**Comment by nejads** [comment]
Done !

**Comment by erictleung** [comment]
@nejads I might have missed something, but it appears you've just resolved the comments without making those changes. Is there are reason for not making those changes without making a comment on those changes?

**Comment by nejads** [comment]
@erictleung Sorry, I have missed it to commit the recommended changes and I don't know why ;). I will do that asap and push it here.

**Comment by nejads** [comment]
@erictleung Sorry, I've missed to making changes. The problem is I opened the pull request, then I managed to accidentally delete the branch I forked and initiated the request from. I would make changes on the PR using this method: 
https://github.com/github/hub/issues/198#issuecomment-147059582
Because I do not have permission to create branches here, I can not go further and I am stuck! 
Do you have any suggestions to solve this problem?

**Comment by raisedadead** [comment]
Hi @nejads,

You have deleted the fork of this repository (not just your branch) as well. At this point the only option is to fork the repo once again and make the changes from scratch.

You can take a look at https://github.com/freeCodeCamp/how-to-contribute-to-open-source/pull/333/files to see what changes were made (and suggested as improvement) and make a fresh PR.

Unfortunately we are going to have to close this.

**Pull request closed without merging**